articles/architecture-scenarios/b2c/user-provisioning.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Consumer Identity and Access Management User Provisioning
+description: Understand how user provisioning works in your CIAM implementation. 
+toc: true
+topics:
+    - b2c
+    - ciam
+    - user-migration
+    - identity-store
+contentType: concept
+useCase:
+  - user-provisioning
+---
+# Consumer Identity and Access Management User Provisioning
+
+User provisioning defines how users will be added to your system. This part of the implementation determines where you will store your users' profile information, how you will migrate existing users, and how your users will sign up. 
+
+## Prerequisites
+
+* Tenant Architecture
+
+## Design considerations
+
+It is important to decide early in your implementation planning how you want users added. The decisions you make influence many other implementation choices.
+
+The factors to consider are: 
+
+1. Where to store your user data, in your own identity data store or in Auth's data store?
+2. If you have legacy user data, do you want to migrate it to Auth0?
+3. Do you want your users to sign up with their existing accounts like Facebook or Google? 
+
+## Identity data storage
+
+During sign up, Auth0 creates a [user profile](/users/concepts/overview-user-profile) that contains information about the user’s account. You can choose to store the user information in Auth0's data store or your own. 
+
+### Use your own identity data store
+
+If you want to use your own identity store, perhaps because you’ve got applications which you aren’t ready to migrate or can’t be migrated, use Auth0’s [Database Connections](/connections/database/custom-db) to proxy an existing identity store. 
+
+### Use Auth0 as the identity data store
+
+Auth0 provides identity data storage to help you manage the burden of storing user credentials safely and securely. If you have a legacy identity store that you would like Auth0 to host, you can use Auth0’s user migration capabilities. 
+
+## User migration
+
+If you decide to use Auth0 instead of your own identity store, you can [migrate users](/users/concepts/overview-user-migration) with bulk migration or progressively with automatic migration. You can also choose a two-phased approach to user migration, using automatic migration first, then using bulk migration for the users that remain. For other scenarios, see [User Migration Scenarios](/users/references/user-migration-scenarios).
+
+### Bulk migration
+
+Auth0 recommends using the [Management API](/users/concepts/overview-user-migration#bulk-user-imports-with-the-management-api) for bulk migration over the [User Import/Export extension](https://auth0.com/docs/users/concepts/overview-user-migration#migrate-users-with-the-user-import-export-extension) in all but the most simple cases. The Management API provides more flexibility and control. 
+
+After the bulk migration is complete, your users will need to reset their passwords.
+
```

Comment:
There are two tools you can use when migrating users in bulk:

1. The Management API
2. The User Import/Export Extension

In most cases, we recommend that you use the Management API, since this option provides you with more flexibility and control. We only recommend the User Import/Export extension in simple cases, such as the movement of a few users.

Please note that your users will need to reset their passwords once you've migrated their accounts to Auth0 using one of the bulk migration processes.